Content/0.4.13 - Device Factor Sources/deviceFactorSources.json

```diff
@@ -0,0 +1,10 @@
+{
+	"deviceFactorSources_navigationTitle": "Biometrics/PIN",
+	"deviceFactorSources_subtitle": "Use phone biometrics/PIN to approve",
+	"deviceFactorSources_default": "Default",
+	"deviceFactorSources_others": "Others",
+	"deviceFactorSources_noProblem": "This seed phrase has been written down",
+	"deviceFactorSources_problem3": "Write down seed phrase to make this factor recoverable",
+	"deviceFactorSources_problem9": "This factor has been lost",
```

Comment:
still not sure if I prefer have these two here or in [securityProblems.json](https://github.com/radixdlt/apps-localization-src/blob/main/en/en-US/Content/Security%20Problems/securityProblems.json) as:
```
"securityProblems_no3_deviceFactorSources": "Write down seed phrase to make this factor recoverable",
"securityProblems_no9_deviceFactorSources": "This factor has been lost"
```

The rationale for adding it here was that we still need to add custom logic (show message for no problem case, different color depending on problem, only show problem for Factor Soruce involved). It is not that we can just do something like `row.problem?.deviceFactorSourcesString` as we do for other scenarios.